.github/CODEOWNERS

```diff
@@ -6,3 +6,5 @@
 /tsdb @jesusvazquez
 /promql @roidelapluie
 /cmd/promtool @dgl
+/documentation/prometheus-mixin @metalmatze
+
```

Comment:
Could we get rid of the space? It does seem to complain

Reply:
this is to be fixed upstream then

Reply:
oh, actually it does seem to complain about ```Unknown owner on line 1: make sure @foo exists and has write access to the repository
```
So I'm wondering whether we should just keep that or override directly.